rush/templates/rush/events.html

```diff
@@ -14,6 +23,72 @@ <h2>Rush Events</h2>
+                            </div>
+                            <div class="modal-body">
+                                <form method="post" action="editEvent{{ event.id }}">
+                                    {% csrf_token %}
+                                    <div class="form-group row">
+                                        <label for="new_name" class="col-sm-2 col-form-label">Name</label>
+                                        <div class="col-sm-10">
+                                            <input type="text" class="form-control" id="edit_name_{{ event.id }}" name="name" value="{{ event.name }}" required>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <label for="new_date" class="col-sm-2 col-form-label">Date</label>
+                                        <div class="col-sm-10">
+                                            <input type="date" class="form-control" id="edit_date_{{ event.id }}" placeholder="YYYY-mm-dd" name="date" value="{{ event.date|date:'Y-m-d' }}" required>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <label for="new_event_round" class="col-sm-2 col-form-label">Round</label>
+                                        <div class="col-sm-10">
+                                            <select class="form-control" id="edit_event_round {{ event.id}}" name="round" value="{{ event.round }}">
+                                                {% for num in round_range %}
+                                                <option>{{ num }}</option>
+                                                {% endfor %}
+                                            </select>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <label for="new_time" class="col-sm-2 col-form-label">Time</label>
+                                        <div class="col-sm-10">
+                                            <input type="time" class="form-control" id="edit_time_{{ event.id }}" name="time" placeholder="HH:mm:ss in 24-hour time" value="{{ event.time|date:'H:i' }}" required>
+                                        </div>
+                                    </div>
+                                    <div class="form-group row">
+                                        <label for="new_location" class="col-sm-2 col-form-label">Location</label>
+                                        <div class="col-sm-10">
+                                            <input type="text" class="form-control" id="edit_location_{{ event.id }}" name="location" value="{{ event.location }}" required>
```

Comment:
In the create form location is not required.  If we want to make it required, we should update it in the create form and the models as well.  I think I would vote it shouldn't be required, in which case we just need to remove the `required` property on this line

Reply:
good catch, agreed